blazor/pager/behaviour-settings.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Behaviour Settings in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about basic functionalites of render the Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Behaviour Settings in Pager Component
+
+## Page size
+
+The Pager component provides an option to define the number of records to be displayed per page. This can be achieved by using the `PageSize` property. The default value of the `PageSize` property is "12".
+
+```csharp
+@using Syncfusion.Blazor.Navigations
+
+<SfPager PageSize="5">
+</SfPager>
+
+```
+
+## Page count
+
+The Pager component provides an option to define the number of numeric items to be displayed in the Pager for navigation. This can be achieved by using the `PageCount` property. The default value of the `PageCount` property is "10".
+
+```csharp
+@using Syncfusion.Blazor.Navigations
+
+<SfPager PageCount="5">
+</SfPager>
+
+```
+
+## Total pages
+
+The Pager component provides an option to define the number of pages available in the Pager. This can be achieved by using the `TotalPages` property. The `TotalPages` value is calculated based on the `TotalItemsCount`.
+
+```csharp
+@using Syncfusion.Blazor.Navigations
+
+<SfPager TotalPages="1">
+</SfPager>
+
+```
+
+## Total items count
+
+The Pager component provides an option to define the total record count, which is used to render the numeric Pager container. This can be achieved by using the `TotalItemsCount` property of the Pager.
```

Comment:
Change the description like below

"The Pager component provides an option to define the total number of items available in the assigned data collection, which is used to render the elements(numeric items, navigation buttons) of Pager component. This can be achieved by using the `TotalItemsCount` property of the Pager."